rskj-core/src/main/java/org/ethereum/net/NodeManager.java

```diff
@@ -45,8 +45,8 @@
 public class NodeManager {
     private static final Logger logger = LoggerFactory.getLogger("discover");
 
-    private static final int MAX_NODES = 2000;
-    private static final int NODES_TRIM_THRESHOLD = 3000;
+    private static final long MAX_NODES = 2000;
```

Comment:
Here I would keep int and wouldn't use protected for NODES_TRIM_THRESHOLD

Reply:
Sonar asked for one of terms of the subtraction to be a long.
I need to access the value of NODES_TRIM_THRESHOLD from test. would you be satisfied with no keyword?